spec/factories/collection_items.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :collection_item do
+    name { 'CollectionItem name' }
+    description { 'This is the description for my CollectionItem' }
+    maker { 'MyMaker' }
+    paid_price { 100.00 }
+    current_value { 200.00 }
+    serial_number { 'Serial1020' }
+    origin { 'MyOrigin' }
+    circa { 2010 }
+    condition { 1 }
```

Comment:
You can use the enum name instead of the integer that is stored on the database.
```suggestion
    condition { : good }
```